geospaas/base_viewer/forms.py

```diff
@@ -0,0 +1,60 @@
+from django.contrib.gis import forms
+from django.utils import timezone
+from django.db.models import Q
+
+from leaflet.forms.widgets import LeafletWidget
+from geospaas.catalog.models import Dataset as CatalogDataset
+from django.contrib.gis.db import models as geomodels
+
+from geospaas.base_viewer.models import Search_dummy_leaflet
+#from geospaas.base_viewer.models import Search_with_para
+class SearchFormBelow(forms.ModelForm):
```

Comment:
If I didn't miss anything, you don't use any functionality of `forms.ModelForm` and could just use `forms.Form`